.vscode/settings.json

```diff
@@ -1,6 +1,3 @@
 {
-    "files.exclude": {
-        ".venv/**": true
```

Comment:
This is no longer necessary after renaming the folder from `venv` to `.venv`

Reply:
because vscode excludes the dot one automatically

Reply:
YES.